### PR TITLE
get last stored value for stash pane width at launch

### LINKED ISSUE #7416: Stash view state is not remembered after restart
As we remember other panes width in Desktop, we should include the stash pane. 

Not sure if the temporary state of this feature will affect this request.

Screen Shot 2019-04-26 at 3 36 56 PM

Repro:
- adjust width
- restart Desktop

Note: we also do not remember that Stash page was last shown. After restarting, the next steps is shown instead of the stash page

1.7.0beta5, mac

### PR DESCRIPTION
**Pull request by outofambit**
## Overview

closes #7416 

## Release notes

Notes: [Fixed] Stash files list retains width between app restarts

### CODE REVIEW

File: app/src/lib/stores/app-store.ts [hunk 1]
```diff
@@ -327,9 +327,18 @@ export class AppStore extends TypedBaseStore<IAppState> {
    */
   private appIsFocused: boolean = false
 
-  private sidebarWidth: number = defaultSidebarWidth
-  private commitSummaryWidth: number = defaultCommitSummaryWidth
-  private stashedFilesWidth: number = defaultStashedFilesWidth
+  private sidebarWidth: number = getNumber(
```
Comment (shiftkey) [inline app/src/lib/stores/app-store.ts]: We already update these values in `loadInitialState()`, and I'm curious whether that isn't necessary any more (tradeoffs aside):

https://github.com/desktop/desktop/blob/be399eb44f3a03ed67fc9ae8b52affdaa4c82f7a/app/src/lib/stores/app-store.ts#L1570-L1574

Alternatively, could we just update `stashedFileChanges` there?

Comment (outofambit) [reply app/src/lib/stores/app-store.ts]: yup, just saw that. totally my mistake. updating now

Comment (outofambit) [reply app/src/lib/stores/app-store.ts]: 🤦‍♀ f5afa9d

**Comment by billygriffin** [comment]
Thanks @outofambit! Just checking, does this also retain knowledge that you were viewing a stash when restarting? That was the other thing @tierninho brought up in the issue.

**Comment by outofambit** [comment]
> does this also retain knowledge that you were viewing a stash when restarting?

@billygriffin no, it does not. i can file a separate issue if we want to discuss/fix that

**Comment by shiftkey** [comment]
> i can file a separate issue if we want to discuss/fix that

👍 we don't remember the tab you were on when you close and restart Desktop, so this is a bit more involved than what's involved with these stashing changes...

**Comment by billygriffin** [comment]
Sounds good to me - I don't have much heartburn about remembering that you were viewing the stash, as it's readily available again for you when you start back up. And it being consistent with other behavior is 👍.

**Comment by tierninho** [comment]
Tested the fix and looks good. 

> Note: we also do not remember that Stash page was last shown. After restarting, the next steps is shown instead of the stash page

We do remember the last repo and last branch when restarting. I thought we got more granular, but no big deal as that is not the case.

**Comment by shiftkey** [comment]
I plan to merge this in later today because the fix has been verified and the change is minor

**Comment by shiftkey** [comment]
> We do remember the last repo and last branch when restarting. 

The last repository is backed by `last-selected-repository-id` in local storage, and the branch is whatever `HEAD` on Git currently points to (so we get that for free).

> I thought we got more granular, but no big deal as that is not the case.

I've not heard many requests for "restore XYZ after closing Desktop" aside from #3664 but happy to discuss options.

**Pull request merged**